docs/outputs.rst

```diff
@@ -22,207 +21,320 @@ FMRIPrep generates three broad classes of outcomes:
      the same-subject's T1w space or into MNI space.
 
   3. **Additional data for subsequent analysis**, for instance the transformations
```

Comment:
Not a big deal, but I expected these three classes to match up their own section below. Maybe highlights the confounds more than the transforms (since those are covered under "derivatives" which in my head matches up with #2 more.